src/DotNetWorker.Core/Diagnostics/TraceConstants.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Functions.Worker.Diagnostics
+{
+    internal class TraceConstants
+    {
+        public const string FunctionsActivitySource = "Microsoft.Azure.Functions.Worker";
+        public const string FunctionsActivitySourceVersion = "1.0.0.0";
+        public const string FunctionsInvokeActivityName = "Invoke";
+
+        public const string AttributeExceptionEventName = "exception";
+        public const string AttributeExceptionType = "exception.type";
+        public const string AttributeExceptionMessage = "exception.message";
+        public const string AttributeExceptionStacktrace = "exception.stacktrace";
+
+        public const string AttributeSchemaUrl = "schema_url";
```

Comment:
nit: we're using `az.schema_url` in Azure SDKs and agreed on it  with azmon folks https://github.com/microsoft/common-schema/blob/main/Mappings/AzureSDKs.md#versioning

Can we align on the name here? I'm open to renaming and generalizing it in MS common-schema beyond azure SDKs

Reply:
Oh yep -- easy to change.